### DISCUSSION TITLE
Node conf wakeUpInterval needs to be a multiple of "STEP" interval

### DISCUSSION THREAD

**Discussion by silabs-theophilel** [zwave-js/zwave-js#7348] [Feature Requests]
### Description

![image](https://github.com/user-attachments/assets/7fff1946-8ecd-4330-b1c7-f5767d82e99f)

In this field, for end device of type Sleeping Reporting a valid configuration needs to meet a few requirements:

MIN < x < MAX
Be a multiple of the STEP value (default value seems to be 20 - at least on [silabs device](https://docs.silabs.com/z-wave/latest/zwave-api/command-class-wake-up-configuration#cc-wake-up-config-sleep-step-time-sec))

### Do you have a solution in mind?
 Values in this field cannot be submitted if it does not match the necessary configuration restrictions.
--> Restrict the possible values

note: this is a specification requirement but the value may be different on platform. So not sure if all platform support the same default value.

### Describe alternatives you've considered

Easier solution: Explicitly mention those requirements with a text message, text will display the configuration of the device: min , max and step values

**Comment by AlCalzone** [comment]
@robertsLando It the UI gets the information which steps are supported, this should be possible to do, right?

**Reply by robertsLando** [reply]
Just added a commit to master to fix this

**Comment by kpine** [comment]
Related: https://github.com/zwave-js/zwave-js-ui/issues/3789

**Comment by robertsLando** [comment]
https://github.com/zwave-js/zwave-js-ui/issues/3980
